Dockerfile

```diff
@@ -6,4 +7,6 @@ RUN apk --update add python py-setuptools py-pip && \
     apk del py-pip && \
     rm -rf /var/cache/apk/*
 
+USER nobody:nobody
```

Comment:
@bobrik This is just an attempt to be safe from situations described in warning block here https://docs.docker.com/engine/reference/builder/#user